db/scripts/sanitise.sql

```diff
@@ -53,26 +42,26 @@ UPDATE equal_opportunities_reports
 
 UPDATE feedbacks
        SET comment=NULL,
-           email=concat('anonymised-feedback-',id,'@example.org'),  -- encrypt. doesn't need id
+           email=concat('anonymised-feedback-',id,'@example.org'),
            other_unsubscribe_reason_comment=NULL,
            visit_purpose_comment=NULL;
 
 UPDATE job_applications
-       SET first_name='Anonymous', -- encrypt
-           last_name='Anon', -- encrypt
-           previous_names='', -- encrypt
-           street_address='1 Example Street', -- encrypt
-           city='Anonymised City', -- encrypt
-           postcode='P05T C0DE', -- encrypt
-           phone_number='01234567890',  -- encrypt
-           teacher_reference_number='1234567',  -- encrypt
-           national_insurance_number='QQ 12 34 56 C', -- encrypt
-           personal_statement='Lorem ipsum dolor sit amet', -- encrypt
-           support_needed_details='', -- encrypt
-           close_relationships_details='', -- encrypt
-           further_instructions='', -- encrypt
-           rejection_reasons='', -- encrypt
-           gaps_in_employment_details='', -- encrypt
+       SET first_name_ciphertext='',
```

Comment:
Is there something I'm missing - shouldn't these be `NULL`?

Reply:
I'll check as it's been a while since I wrote this

Reply:
They work either way. I don't see a reason to prefer NULL to `''`.

Reply:
@csutter I'll wait to hear your reply tomorrow and leave this merge until then.

Reply:
I would say `NULL` is more obvious in what it represents - absence of value rather than the value of "empty string". That said, given what these are (ciphertext columns) it should be obvious either way, so happy to keep it at `''`.